packages/core/FieldArray.h

```diff
@@ -160,7 +160,8 @@ FieldArray<T,N>::FieldArray(std::initializer_list<T> init_list):
  *----------------------------------------------------------------------------*/
 template <class T, int N>
 FieldArray<T,N>::FieldArray(void):
-    FieldUnsafeArray<T>(&values[0], N)
+    FieldUnsafeArray<T>(&values[0], N),
+    values()
```

Comment:
I prefer to leave a value uninitialized when it is actually uninitialized.  This allows address sanitizer / valgrind to catch uninitialized data access.